src/datastore/api.py

```diff
@@ -147,9 +147,14 @@ def get_copernicus_brasil(
     tags=["datastore", "contaovos"],
 )
 @csrf_exempt
-def get_contaovos(request, key: str, page: int):
-    url = "https://contaovos.dengue.mat.br/pt-br/api/lastcounting"
-    params = {"key": key, "page": page}
+def get_contaovos(
```

Comment:
Is it possible to add start and end dates to the request?

Reply:
Only if the start and and dates are filters in the contaovos API, otherwise would be impossible to the filter know how many results exist in the data for that specific date range

Reply:
Leon said that if we need any change to the API he can do it, so we can ask him to add the possibility to filter by date